src/Turtle/Prelude.hs

```diff
@@ -228,6 +228,29 @@ module Turtle.Prelude (
     , gibibytes
     , tebibytes
 
+    -- * File status
+    , PosixCompat.FileStatus
+    , stat
+    , lstat
+    , PosixCompat.deviceID
+    , PosixCompat.fileID
+    , PosixCompat.fileMode
+    , PosixCompat.linkCount
+    , PosixCompat.fileOwner
+    , PosixCompat.fileGroup
+    , PosixCompat.specialDeviceID
+    , PosixCompat.fileSize
+    , PosixCompat.accessTime
+    , PosixCompat.modificationTime
```

Comment:
For `stat`-related functions that return an `EpochTime` like `accessTime` and `modificationTime`, could we wrap them to return a `time` library type like `PosixTime`?